lib/cli/util.js

```diff
@@ -40,6 +40,12 @@ module.exports = {
       } else {
         const filePath = path.resolve(url);
         try {
+          // Check extention file is js
+          if (path.extname(filePath) === '.js') {
```

Comment:
Think this will break for people that do not use extension `.js.` At the moment I think we can have whatever extension?